sklearn/linear_model/coordinate_descent.py

```diff
@@ -69,6 +69,11 @@ def _alpha_grid(X, y, Xy=None, l1_ratio=1.0, fit_intercept=True,
     copy_X : boolean, optional, default True
         If ``True``, X will be copied; else, it may be overwritten.
     """
+    if np.unique(y).size == 1:
```

Comment:
If we hard-code this case, we could always just create a DummyRegressor and save a lot of work ;) I guess it would make the code ugly, though.
Maybe we should warn here.